src/client/datascience/notebook/notebookControllerManager.ts

```diff
@@ -549,4 +557,32 @@ export class NotebookControllerManager implements INotebookControllerManager, IE
             controller.dispose();
         });
     }
+
+    // Load up a python kernel to just launch in the current vscode environment
+    private loadCurrentPythonEnvironmentKernel() {
+        const config = this.configuration.getSettings();
```

Comment:
Do we want this kernel to be the preferred kernel for all new vs code users?
Or should that be in an experiment.

I feel that even if we merged this in, very very few would actually use this
& I think the new users are the ones to benefit the most from this (but only via an experiment, else its possible).